src/bulk/common.ts

```diff
@@ -1011,68 +1032,57 @@ class BulkOperationBase {
     return new FindOperators(this);
   }
 
-  /**
-   * Specifies a raw operation to perform in the bulk write.
-   *
-   * @function
-   * @param op The raw operation to perform.
-   * @param reference for more information.
-   * @returns {BulkOperationBase} A reference to self
-   */
-  raw(op: any): BulkOperationBase {
-    const key = Object.keys(op)[0];
-
+  raw(op: Partial<BatchOperations> & { collation?: CollationOptions }): BulkOperationBase {
```

Comment:
Does an optional collation not fit (as in cause type issues?) on the BatchOperations interface?